datamodules/lucas_processing.py

```diff
@@ -60,10 +65,18 @@ def __init__(
                     "Must provide either end_cutoff or end_cutoff_timesteps"
                 )
 
-            d = torch.tensor(shot_df[:shot_end])
+            timedelta_columns = [col for col in shot_df.columns if isinstance(shot_df[col].iloc[0], pd.Timedelta)]
```

Comment:
Did you observe this in the version of the dataset that gets downloaded?

Reply:
no, I didn't